smart-contracts/test/helpers/proxy.js

```diff
@@ -0,0 +1,26 @@
+const Zos = require('zos')
+const { ZosPackageFile } = Zos.files
+const packageFile = new ZosPackageFile()
+
+let networkFile
+let proxiesArray
+let proxies
+let mostRecentProxy
+let ProxyAddress
+let proxiedUnlock
+
+module.exports = function setUnlockProxy (_Unlock, _network) {
```

Comment:
minor, but would it be more clear to call this `getUnlockProxy`?  It doesn't actually deploy or change anything.

Reply:
I think you're right.